admin/views/tabs/social/facebook.php

```diff
@@ -7,87 +7,49 @@
 	);
-	// phpcs:ignore WordPress.Security.EscapeOutput -- get_button_html() output is properly escaped.
-	echo '<h2 class="help-button-inline">' . esc_html__( 'Frontpage settings', 'wordpress-seo' ) . $social_facebook_frontpage_help->get_button_html() . '</h2>';
-	// phpcs:ignore WordPress.Security.EscapeOutput -- get_panel_html() output is properly escaped.
-	echo $social_facebook_frontpage_help->get_panel_html();
-
-	$yform->hidden( 'og_frontpage_image', 'og_frontpage_image' );
-	$yform->hidden( 'og_frontpage_image_id', 'og_frontpage_image_id' );
-	?>
-	<div id="yoast-og-frontpage-image-select"></div>
-	<?php
-	$yform->textinput( 'og_frontpage_title', __( 'Title', 'wordpress-seo' ) );
-	$yform->textinput( 'og_frontpage_desc', __( 'Description', 'wordpress-seo' ) );
 
-	$copy_home_description_button_label = esc_html__( 'Copy home meta description', 'wordpress-seo' );
-
-	// Offer copying of meta description.
-	$homepage_meta_description = WPSEO_Options::get( 'metadesc-home-wpseo' );
-	if ( ! empty( $homepage_meta_description ) ) {
-		$copy_home_meta_desc_help = new WPSEO_Admin_Help_Panel(
-			'copy-home-meda-desc',
-			esc_html__( 'Help on copying the home meta description', 'wordpress-seo' ),
-			sprintf(
-				/* translators: 1: link open tag; 2: link close tag., 3: the translated label of the button */
-				esc_html__( 'Click the "%3$s" button to use the meta description already set in the %1$sSearch Appearance Homepage%2$s setting.', 'wordpress-seo' ),
-				'<a href="' . esc_url( admin_url( 'admin.php?page=wpseo_titles' ) ) . '">',
-				'</a>',
-				// $copy_home_description_button_label is escaped above.
-				$copy_home_description_button_label
-			)
-		);
-
-		echo '<input type="hidden" id="meta_description" value="', esc_attr( $homepage_meta_description ), '" />';
-		echo '<div class="label desc copy-home-meta-description">' .
-			// phpcs:ignore WordPress.Security.EscapeOutput -- $copy_home_description_button_label is escaped above.
-			'<button type="button" id="copy-home-meta-description" class="button">', $copy_home_description_button_label, '</button>' .
-			// phpcs:ignore WordPress.Security.EscapeOutput -- get_button_html() output is properly escaped.
-			$copy_home_meta_desc_help->get_button_html() .
-			// phpcs:ignore WordPress.Security.EscapeOutput -- get_panel_html() output is properly escaped.
-			$copy_home_meta_desc_help->get_panel_html() .
-			'</div>';
-	}
-}
+	$frontpage_settings_alert = new Alert_Presenter( $frontpage_settings_message, 'info' );
 
-echo '<h2>' . esc_html__( 'Default settings', 'wordpress-seo' ) . '</h2>';
+	// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- Output from present() is considered safe.
+	echo '<div class="yoast-measure">' . $frontpage_settings_alert->present() . '</div>';
 
-$yform->hidden( 'og_default_image', 'og_default_image' );
-$yform->hidden( 'og_default_image_id', 'og_default_image_id' );
+	echo '<h2>' . \esc_html__( 'Default settings', 'wordpress-seo' ) . '</h2>';
```

Comment:
I see that in the sketch, the design for the image default setting is different. Screenshot:

<img width="828" alt="Screenshot 2021-04-13 at 15 27 51" src="https://user-images.githubusercontent.com/1682452/114560459-012c9780-9c6d-11eb-8db5-fef19890df5c.png">